pkg/cloud-provider/cloudapi/aws/aws_security.go

```diff
@@ -535,6 +536,8 @@ func (ec2Cfg *ec2ServiceConfig) getNepheControllerManagedSecurityGroupsCloudView
 					Name: networkInterfaceID,
 					Vpc:  *networkInterface.VpcId,
 				},
+				AccountID:     ec2Cfg.accountName,
+				CloudProvider: string(v1alpha1.AWSCloudProvider),
```

Comment:
Do you need CloudProvider, as you are directly accessing that namespaced-name map